ext/enchant/tests/invalidobj.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+invalid object raise exception() function
+--SKIPIF--
+<?php
+if(!extension_loaded('enchant')) die('skip, enchant not loader');
+if (!is_object(enchant_broker_init())) {die("skip, resource dont load\n");}
+?>
+--FILE--
+<?php
+$broker = enchant_broker_init();
+if (is_object($broker)) {
+    echo "OK\n";
+    @enchant_broker_free($broker);
+	try {
```

Comment:
nit: Something is off about indentation here.

Reply:
fixed